utils/src/main/scala/it/pagopa/interop/commons/utils/package.scala

```diff
@@ -23,6 +23,9 @@ package object utils {
   val DIGEST_CLAIM: String                                 = "digest"
   val ORGANIZATION_ID_CLAIM: String                        = "organizationId"
   val SELFCARE_ID_CLAIM: String                            = "selfcareId"
+  val EXTERNAL_ID_CLAIM: String                            = "externalId"
+  val ORIGIN: String                                       = "origin"
+  val VALUE: String                                        = "value"
```

Comment:
In order to be globally clear, I suggest these values
```suggestion
  val ORGANIZATION_EXTERNAL_ID_CLAIM: String                            = "externalId"
  val ORGANIZATION_EXTERNAL_ID_ORIGIN: String                           = "origin"
  val ORGANIZATION_EXTERNAL_ID_VALUE: String                            = "value"
```